api/tests/core/bookings/test_api.py

```diff
@@ -836,35 +836,31 @@ def test_raise_if_booking_was_automatically_used_for_digital_offer_with_activati
 
 
 @pytest.mark.parametrize(
-    "booking_date",
+    "booking_creation",
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:** `‏booking_creation_date` ? 
looow logaf